modules/navier_stokes/test/tests/finite_volume/wcns/materials/enthalpy_computation.i

```diff
@@ -0,0 +1,101 @@
+[Mesh]
```

Comment:
What is the purpose of this test? Let's compute the enthalpies by hand and compare the code result against the hand calculation. That way we know that things are being done properly

Reply:
it's the original regression test. I ll add the hand calc

Reply:
done. Had to change to PbBiFPs because Water97 is quite complex in terms of computing a specific fluid property